backend/src/api/routes/database/index.routes.ts

```
@@ -108,4 +108,67 @@ router.get(
   }
 );
 
+/**
+ * Create a new index.
+ * POST /api/database/indexes
+ *
+ * Validates for name collisions and duplicate column coverage, then builds
+ * the index synchronously. For CONCURRENTLY builds on large tables this
+ * request may take several minutes — the client should show a loading state.
+ */
+router.post('/indexes', verifyAdmin, async (req: AuthRequest, res: Response, next: NextFunction) => {
```

ESLint

[error] Replace `'/indexes',·verifyAdmin,` with `⏎··'/indexes',⏎··verifyAdmin,⏎·`

(prettier/prettier)